netbox/netbox/views/generic/bulk_views.py

```diff
@@ -503,6 +503,21 @@ def _update_objects(self, form, request):
         ]
         nullified_fields = request.POST.getlist('_nullify')
         updated_objects = []
+        model_fields = {}
+        m2m_fields = {}
+
+        # Update standard fields. If a field is listed in _nullify, delete its value.
```

Comment:
Don't forget to update the comments appropriately. (This has been duplicated from below.)

Reply:
Fixed up that comment, whoops